### PR TITLE
docs: replace npm with yarn in lockfile readme

### PR DESCRIPTION
**Pull request by DanielRuf**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

npm is still mentioned in the readme of the lockfile.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: yarn.lock.readme.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 All of our npm dependencies are locked via the `yarn.lock` file for the following reasons:
 
 - our project has lots of dependencies which update at unpredictable times, so it's important that
-  we update them explicitly once in a while rather than implicitly when any of us runs npm install
-- locked dependencies allow us to reuse npm cache on travis, significantly speeding up our builds
+  we update them explicitly once in a while rather than implicitly when any of us runs `yarn install`
```
Comment (brandonroberts) [inline yarn.lock.readme.md]: This can just be `yarn` instead of `yarn install`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: yarn.lock.readme.md (hunk 1 above)
Comment (DanielRuf) [reply yarn.lock.readme.md]: Not really, you can use both ;-)

Comment (DanielRuf) [reply yarn.lock.readme.md]: See https://yarnpkg.com/en/docs/cli/install

> Running yarn with no command will run yarn install, passing through any provided flags.

Comment (brandonroberts) [reply yarn.lock.readme.md]: That's correct, but if there is no difference, why type the extra `install`?

Comment (brandonroberts) [reply yarn.lock.readme.md]: For consistency with the developer doc, `yarn install` is fine.

**Review by brandonroberts — APPROVED** [review]

File: yarn.lock.readme.md (hunk 1 above)
Comment (DanielRuf) [reply yarn.lock.readme.md]: > That's correct, but if there is no difference, why type the extra install

To ease the migration for developers who know npm (a bit DX) ;-)

But you are right, we can also change this to `yarn`.

Shall I do it and remove `install` from the command?

Comment (brandonroberts) [reply yarn.lock.readme.md]: Yes

Comment (gkalpak) [reply yarn.lock.readme.md]: FWIW, I think `yarn install` is better, as it is clear even for people that are not familiar with `yarn`.
(I.e. in the same way that we didn't use `npm i` but `npm install` before.)

Comment (brandonroberts) [reply yarn.lock.readme.md]: This is not a hill I'm going to die on :-) I'm fine with `yarn install` if we're targeting new people.

Comment (gkalpak) [reply yarn.lock.readme.md]: ⚔️

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**